easybuild/easyblocks/n/numpy.py

```diff
@@ -214,9 +253,22 @@ def get_libs_for_mkl(varname):
 
     def test_step(self):
         """Run available numpy unit tests, and more."""
-        super(EB_numpy, self).test_step()
 
-        # temporarily install numpy, it doesn't alow to be used straight from the source dir
+        if self.cfg['runtest']:
+            # Let's handle the output from the numpy test suite ourselves
+            testcmd_output, testcmd_exit_code = super(EB_numpy, self).test_step(return_testcmd_output=True)
+
+            testsuite_summary = parse_numpy_test_suite_output(testcmd_output)
+
+            if testsuite_summary and ('failed' in testsuite_summary or 'error' in testsuite_summary):
```

Comment:
Does this handle all types of return values from parse_numpy_test_suite_output?
res = summary_regex.search
res = None
res = dict

Reply:
Yes, `summary_regex.search` only get's returned if it's 'truthiness' is false, `None` is also false, and the dict is explicitly handled.
I can make `parse_numpy_test_suite_output` explicitly return False for the first two cases though, just to very clear on the return possibilities.

Reply:
Returning only a dict or `None` made more sense so I've implemented that